doc: TF-M updates for the NCS 2.0.0 release (#7614)
nrfconnect/sdk-nrf: doc/nrf/ug_nrf91_features.rst

A bootloader chain is optional.
Not all of the nRF9160 samples include a secure bootloader chain, but the ones that do use the :ref:`bootloader` sample and :doc:`mcuboot:index-ncs`.

Trusted Firmware-M (TF-M)
-------------------------

Trusted Firmware-M provides a configurable set of software components to create a Trusted Execution Environment.

> It is replacing Secure Partition Manager in being the default solution used by most |NCS| applications and samples.

Reviewer: ```suggestion
It is replacing Secure Partition Manager as the default solution used by most |NCS| applications and samples.
```
Maybe also use "has replaced" so we don't need to change the tense in future releases?

> This means that when you build your application for build targets ``nrf9160dk_nrf9160_ns`` and ``thingy91_nrf9160_ns``, the TF-M is automatically included in the build.

Reviewer 2: Maybe use "build targets ``_ns`` " instead of listing the boards. There can be other boards in addition, as well as the users own board definitions.
